pre_commit/util.py

```diff
@@ -268,3 +271,70 @@ def handle_remove_readonly(
 def parse_version(s: str) -> Tuple[int, ...]:
     """poor man's version comparison"""
     return tuple(int(p) for p in s.split('.'))
+
+
+def in_docker() -> bool:
+    """
+    Check if running in Docker
+    :return: Whether or not this is running in Docker container
+    """
```

Comment:
this docstring is incredibly useless!  what else would an `in_docker` function do?